semanticdb/metacp/src/main/scala/scala/meta/internal/classpath/ClasspathIndex.scala

```diff
@@ -40,6 +36,9 @@ final class ClasspathIndex private (val dirs: collection.Map[String, Classdir])
   def isClassdir(path: String): Boolean =
     dirs.contains(path)
 
+  override def toString: String = {
+    s"ClasspathIndex($classpath)"
```

Comment:
Is this helpful? In my experience the full classpath is too verbose and is not helpful in debugging a problem. I would prefer to expose the `Classpath` so that users can drill into `.entries` to investigate.

Reply:
That was helpful to debug what classpath Scalafix receives from our Pants integration. Exposing classpath would've achieved the same goal, so I'm totally up for that.